packages/manager/modules/pci/src/projects/project/training/jobs/translations/Messages_fr_FR.json

```diff
@@ -44,9 +44,11 @@
   "pci_projects_project_training_jobs_submit_data_prefix": "Filtrer par préfixe",
   "pci_projects_project_training_jobs_submit_data_prefix_help": "Seuls les objets correspondants au préfixe sont chargés dans le système de fichier",
   "pci_projects_project_training_jobs_submit_data_permission": "Permission",
+  "pci_projects_project_training_jobs_submit_data_permission_select": "Permission",
   "pci_projects_project_training_jobs_submit_data_permission_help": "En `Lecture Seule` aucune synchronisation n'est lancée en fin de job. Le job se finalise ainsi plus rapidement.",
   "pci_projects_project_training_jobs_submit_data_permission_RO": "Lecture Seule",
   "pci_projects_project_training_jobs_submit_data_permission_RW": "Lecture & Ecriture",
+  "pci_projects_project_training_jobs_submit_data_permission_RWD": "Lecture & Ecriture & Suppression",
```

Comment:
For me RW means "Read and Write" which allow reading, writing and also deleting.
Regarding UNIX permissions we have only r, w and x.

What do you think ?

Reply:
That is true and a tad confusing, but that is the values we have on the backend side right now, so we cannot change this